portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,28 +32,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> comments = new ArrayList<>();
+  private List<String> comments;
   private Gson gson = new Gson();
+  private static final String COMMENT = "Comment";
+  private static final String TEXT = "text";
```

Comment:
These names are not good enough. The fields don't represent comment or text content, but the column/field names when you interact with DataStore. The followings are better names:

COMMENT_COLUMN
COMMENT_FIELD
COMMENT_COLUMN_NAME
COMMENT_FIELD_NAME